portfolio/src/main/webapp/index.html

```diff
@@ -1,19 +1,46 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <style>
+        body {
+        background-color:powderblue;
+        }
+
+        h1 {
+        color: white;
+        font-family: "Lucida Console", "Courier New", monospace;
+        text-align: center;
+        }
+
+        p {
+        color: whitesmoke;
+        font-family: "Lucida Console", "Courier New", monospace;
+        font-size: 20px;
+        }
+        
+    </style>
```

Comment:
You can put all this styling in portfolio/src/main/webapp/style.css to keep this organized